src/modules/orderHelper.js

```diff
@@ -4,27 +4,28 @@ const authHeaders = JSON.parse(localStorage.getItem('userData'))
 
 const createOrder = async (item_id) => {
   let response = await axios.post(
-    './orders',
-     {item_id: item_id}, 
-     {headers: authHeaders}) 
+    'http://localhost:3000/api/orders',
+    {menu_item_id: item_id}, 
+     {headers: authHeaders})
+     
   return response.data
 }
 
 const updateOrder = async (item_id, order_id) => {
   let response = await axios.put(
-    `./orders/${order_id}`,
-     {item_id: item_id}, 
+    `http://localhost:3000/api/orders/${order_id}`,
+     {menu_item_id: item_id}, 
      {headers: authHeaders}) 
-  return response.data.body
+  return response.data
 }
 
 const getOrder = async (order_id) => {
-  const response = await axios.get(`/orders/${order_id}`, {headers: authHeaders})
-  return response.data.order.items;
+  const response = await axios.get(`http://localhost:3000/api/orders/${order_id}`, {headers: authHeaders})
+  return response.data.order.menu_items;
```

Comment:
why did it become menu_items?

Reply:
because this is the correct key that backend is sending